nixos/modules/system/boot/uki.nix

```diff
@@ -51,16 +51,16 @@ in
     else
       "nixos");
 
-    boot.uki.settings = lib.mkOptionDefault {
+    boot.uki.settings = {
```

Comment:
So if we keep the  `mkOptionDefault` for the attrset, the overriding wouldn't work correctly anymore?

Reply:
correct, with 
```
boot.uki.settings = lib.mkOptionDefault { UKI = { ... }; };
```
here in the module and the nixos configuration
```
boot.uki.settings.UKI.Uname = "foobar";
```
following ukify config is the result:
```ini
[UKI]
Uname=foobar
```